data/json/npcs/godco/NPC_Helena_Misinter.json

```diff
@@ -0,0 +1,280 @@
+          { "u_add_var": "godco_machine_reward", "type": "mission", "context": "reward", "value": "icon" },
+          { "u_add_var": "unclaimed_machine_reward", "type": "mission", "context": "reward", "value": "yes" },
+          { "u_spawn_item": "icon", "count": 25 }
+        ],
+        "topic": "TALK_DONE"
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_machineFood",
+    "dynamic_line": "A generous soul, are we?  That's not what I was expecting, but I'll see to it.",
+    "responses": [
+      {
+        "text": "Thank you.",
+        "effect": [
+          { "u_lose_var": "godco_corrie_machine", "type": "mission", "context": "completed" },
+          { "u_add_var": "godco_machine_reward", "type": "mission", "context": "reward", "value": "ration" },
+          { "u_add_var": "unclaimed_machine_reward", "type": "mission", "context": "reward", "value": "yes" }
+        ],
+        "topic": "TALK_DONE"
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_Contact_FreeMerch",
+    "dynamic_line": "Yes, I was just informed of that.  I also heard that she found a large group of people living out in an old evacuation center, who consider themselves merchants.  That would be a good place to resume our spread of the Truth, perhaps we could find others who were faithful before <the_cataclysm>.",
+    "responses": [
+      {
+        "text": "I presume you want me to pay a visit?",
+        "condition": {
+          "and": [
+            { "not": { "u_has_var": "u_met_Gavin", "type": "general", "context": "meeting", "value": "yes" } },
+            { "not": { "u_has_var": "found_refugee_center", "value": "yes" } }
+          ]
+        },
+        "topic": "TALK_GODCO_Helena_Accept_Contact_FreeMerch"
+      },
+      {
+        "text": "Oh, I've met them before!",
+        "condition": {
+          "or": [
+            { "u_has_var": "u_met_Gavin", "type": "general", "context": "meeting", "value": "yes" },
+            { "u_has_var": "found_refugee_center", "value": "yes" }
+          ]
+        },
+        "topic": "TALK_GODCO_Helena_Accept_Contact_FreeMerch"
+      },
+      { "text": "Let's change the subject.", "topic": "TALK_GODCO_Helena_2" },
+      { "text": "Good luck with that.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_Accept_Contact_FreeMerch",
+    "dynamic_line": "Yes, you seem like a good fit for the job.  I'll pay you a small sum for getting some information about them.",
+    "responses": [
+      {
+        "text": "Alright, I'll go and see what's up.",
+        "effect": { "add_mission": "MISSION_GODCO_HELENA_VISIT_MERCHANTS" },
+        "topic": "TALK_DONE"
+      },
+      { "text": "Let's change the subject.", "topic": "TALK_GODCO_Helena_2" },
+      { "text": "Good luck with that.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
```

Comment:
I'd like to see conflict between the Norse Pagans and Godco.  It doesn't have to be in this PR but I'd like a comment `"//"`  showing it as a long term goal possibly?

Reply:
With the hordes of zombies roaming the countryside and the general lack of basic supplies, the New England Church Community and the Norse Evangelicals likely wouldn't care about each other unless one goes on the offensive. The leadership of both factions is smart enough to realize a war between the two would be a waste of time, manpower and materials. They also share quite a few goals, including the acquisition of resources and the elimination of the Mycus, zombies, Triffids, etc.
If so inclined, it would be interesting to the player start a war between them. In most all situations of that nature, the Norse Evangelicals would win; they're trained warriors in melee and ranged combat, and properly armored and equipped. Godco only has 1-2 people who can fight reliably, most are average people.

Reply:
I'll put that in.